microsoft-edge/devtools-guide-chromium/whats-new/2021/05/devtools.md

```diff
@@ -0,0 +1,228 @@
+
+:::row:::
+    :::column:::
+        :::image type="complex" source="../../media/2021/05/screen-reader-support-in-console-autocomplete.msft.png" alt-text="In the Console, screen readers now announce the currently selected autocomplete suggestion" lightbox="../../media/2021/05/screen-reader-support-in-console-autocomplete.msft.png":::
+           In the **Console**, screen readers now announce the currently selected autocomplete suggestion
+        :::image-end:::
+    :::column-end:::
+    :::column:::
+        :::image type="complex" source="../../media/2021/05/screen-reader-support-in-console-evaluated-expression.msft.png" alt-text="In the Console, screen readers now announce the result of an evaluated expression" lightbox="../../media/2021/05/screen-reader-support-in-console-evaluated-expression.msft.png":::
+           In the **Console**, screen readers now announce the result of an evaluated expression
+        :::image-end:::
+    :::column-end:::
+:::row-end:::
+        
+For a related feature in Microsoft Edge version 89, navigate to [Assistive technologies now announce position and count of CSS suggestions][DevtoolsWhatsnew202101DevtoolsAssistiveAnnounce].
+
+
+## Microsoft Edge Developer Tools for Visual Studio Code version 1.1.8
+
+The [Microsoft Edge Developer Tools for Visual Studio Code][VisualstudioMarketplaceMsEdgedevtoolsVscodeEdgeDevtools] extension version 1.1.8 for Microsoft Visual Studio Code has the following changes since the previous release.  Microsoft Visual Studio Code updates extensions automatically.  To manually update to version 1.1.8, navigate to [Update an extension manually][VisualstudioCodeDocsEditorExtensionGalleryUpdateExtensionManually].  
+
+You can file issues and contribute to the extension on the [vscode-edge-devtools GitHub repo][GithubMicrosoftVscodeEdgeDevtools].  
+
+### In-context documentation and UI to make it easier to use the DevTools extension
+
+<!-- Title: In-context documentation and UI make it easier to get started using the Developer Tools extension -->  
+<!-- Subtitle: The Microsoft Edge Developer Tools for Visual Studio Code extension now presents helpful text, buttons, and links, and opens a documentation page with guidance on how to get started. -->  
+
+Version 1.1.8 of the [Microsoft Edge Developer Tools for Visual Studio Code][VisualstudioMarketplaceMsEdgedevtoolsVscodeEdgeDevtools] extension now features a simpler way to start a new instance of Microsoft Edge, by presenting instructions, buttons, links, and a documentation page to guide you.
+
+*  When you select the **Microsoft Edge Tools** button in the **Activity Bar** on the left side of Visual Studio Code, the **Microsoft Edge Tools: Targets** panel now presents explanatory text, buttons, and links to guide you, instead of a blank panel.
```

Comment:
```suggestion
*  When you select the **Microsoft Edge Tools** button in the **Activity Bar** of Visual Studio Code, the **Microsoft Edge Tools: Targets** panel now presents explanatory text, buttons, and links to guide you, instead of a blank panel.
```